docs/build-on-btc/using-regtest.mdx

```diff
@@ -0,0 +1,171 @@
+---
+keywords: [bitcoin, btc dev, btc dev environment, developer environment for btc, dev env btc, setup environment, btc environment, deploy bitcoin locally, local bitcoin, local bitcoin development, develop environment, development environment, regtest testing, regtest network, local testnet, regression testing, btc regtest]
+---
+
+import TabItem from "@theme/TabItem";
+import { AdornedTabs } from "/src/components/Tabs/AdornedTabs";
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import { OsType, useOs } from "/src/hooks/useOs";
+
+# Using Bitcoin regtest
+
+<MarkdownChipRow labels={["Bitcoin"]} />
+
+Before using the local Bitcoin regtest instance, you will need to:
+
+- [x] [Setup the local developer environment](/docs/build-on-btc/btc-dev-env).
+
+- [x] Clone the `@dfinity/examples` repo: `git clone https://github.com/dfinity/examples.git`
+
+This page will demonstrate how to use the local Bitcoin regtest instance using the [`basic_bitcoin` example](https://github.com/dfinity/examples/tree/master/rust/basic_bitcoin) project written in Rust. Alternatively, you can use the Motoko variation of this project.
+
```

Comment:
@jessiemongeon1 There is one additional thing you need to do on Mac to get the example to build. I describe this in the readme of that project. 

- [x] On macOS, an `llvm` version that supports the `wasm32-unknown-unknown` target is required. This is because the Rust `bitcoin` library relies on `secp256k1-sys`, which requires `llvm` to build. The default `llvm` version provided by XCode does not meet this requirement. Instead, install the [Homebrew version](https://formulae.brew.sh/formula/llvm), using `brew install llvm`. 

Perhaps we should have that info here as well?